compiler/parser/src/python.rs

```diff
@@ -1,3 +1,3 @@
 #![allow(clippy::all)]
 #![allow(unused)]
-include!(concat!(env!("OUT_DIR"), "/python.rs"));
+include!("../python.rs");
```

Comment:
could you give more context behind this?

Reply:
In [our environment](https://internetcomputer.org/) which is a decentralized Wasm cloud environment, there are function complexity checks that are performed before a Wasm binary is allowed to be deployed. The complexity limit is currently `15_000`, but the generated lolrpop python.rs file has a function called `__reduce` that is ~9000 lines long and has a complexity of `16_722`. So for now we need to modify the generated python.rs file and split up the `__reduce` function. We have asked the protocol engineers of the Wasm environment to increase the complexity limit, but if you know of how to get lolrpop to generate less complex functions that would be great too.

Reply:
> but if you know of how to get lolrpop to generate less complex functions that would be great too.

Doesn't seem likely https://github.com/lalrpop/lalrpop/issues/304. Hopefully at some point we should get a PEG parser which shouldn't result in such a behemoth of a function.

Reply:
Idk if it is possible now, but if you can take input as `*.pyc` instead of `*.py`, skipping parsing from the embedded machine will be possible (maybe with little bit fix of rustphython).